.github/workflows/lint-build-test.yml

```
@@ -13,19 +13,19 @@ jobs:
       matrix:
         node-version: [14.x, 16.x]
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
       - name: Get Yarn cache directory
-        run: echo "::set-output name=YARN_CACHE_DIR::$(yarn config get cacheFolder)"
+        run: echo "YARN_CACHE_DIR=$(yarn config get cacheFolder)" >> $GITHUB_OUTPUT
```

These should be quoted.

```suggestion
        run: echo "YARN_CACHE_DIR=$(yarn config get cacheFolder)" >> "$GITHUB_OUTPUT"
```

Maybe add `actionlint` to this PR, so we catch these things?

---

Yup, I can add `actionlint`. In terms of caching it looks like the `setup-node` action [already supports caching Yarn dependencies](https://github.com/actions/setup-node#caching-global-packages-data), so I don't think we even need this here.

---

`actionlint` added here: [cc9bdbb](https://github.com/MetaMask/controllers/pull/831/commits/cc9bdbb00dd0a5fc547273372dfb8fc2e5c60185).